x/nodes/keeper/reward_test.go

```diff
@@ -87,6 +90,28 @@ func TestMint(t *testing.T) {
 	}
 }
 
+func verifyAccountBalance(
+	t *testing.T,
+	k Keeper,
+	ctx sdk.Context,
+	address sdk.Address,
+	expected sdk.BigInt,
+) {
+	acc := k.GetAccount(ctx, address)
+	expectedCoins := sdk.NewCoins(sdk.NewCoin("upokt", expected))
+	assert.True(t, acc.Coins.IsEqual(expectedCoins))
+	if !acc.Coins.IsEqual(expectedCoins) {
+		fmt.Println(
+			"Balance mismatch",
+			address,
+			"actual=",
+			acc.Coins,
+			"expected=",
+			expectedCoins,
+		)
+	}
```

Comment:
For 1), it does.  Assert doesn't stop the test.

I agree with 2) that looks smarter.  Will change it.  Practically, however, having this kind of false condition is much easier to debug because we can set a breakpoint there.